src/e2e/app.ts

```diff
@@ -107,6 +107,17 @@ export class App {
       await dialog.accept();
     });
 
+    const logsPath =
+      "reports/screenshots/" + expect.getState().currentTestName + ".txt";
```

Comment:
I think it'd now make more sense if "screenshots" was be e.g. "e2e" here and below (ideally using a common constant).

Reply:
[`109f052`](https://github.com/microbit-foundation/python-editor-next/pull/474/commits/109f052b09bee1277305477bdc62a428562bf793)